### PR TITLE
Add api documentation link in project dashboard

### PR DESCRIPTION
**Pull request by pranavxc**
Added data APIs documentation in project dashboard navdrawer.

### CODE REVIEW

File: packages/nc-gui/components/project/spreadsheet/components/extras.vue [hunk 1]
```diff
@@ -29,65 +30,86 @@
           :class="{ active: true }"
           dense
         >
-          <v-list-item dense href="https://discord.gg/5RgZmkW" target="_blank">
-            <!-- Join Discord -->
+          <v-list-item>
+            <div class="d-flex justify-space-between d-100 pr-2">
+              <v-icon v-t="['community:discord']" class="mr-1" size="22" :color="textColors[0]" @click="open('https://discord.gg/5RgZmkW','_blank')">
```
Comment (wingkwong) [inline packages/nc-gui/components/project/spreadsheet/components/extras.vue:35]: I think the second parameter is not necessary as the default target in `open()` is `_blank` already.

Comment (pranavxc) [reply packages/nc-gui/components/project/spreadsheet/components/extras.vue:35]: ya true, i will update

File: packages/nc-gui/components/ProjectTreeView.vue [hunk 2]
```diff
@@ -593,6 +593,17 @@
             </settings-modal>
           </div>
         </template>
+
+        <div
+          v-t="['api-docs']"
+          class="caption pointer nc-team-settings pb-3 pl-5 pr-3 pt-2 d-flex align-center"
+          @click="openLink('https://apis.nocodb.com')"
```
Comment (wingkwong) [inline packages/nc-gui/components/ProjectTreeView.vue:600]: Should we show the all-apis one? Some users may be interested in APIs like how to create a table.

Comment (o1lab) [reply packages/nc-gui/components/ProjectTreeView.vue:600]: Lets point it to apis.nocodb for now.

All APIs will be overwhelming for user who doesnt need the meta apis

File: packages/nc-gui/components/project/spreadsheet/components/extras.vue [hunk 3]
```diff
@@ -32,16 +32,16 @@
...
               </v-icon>
-              <v-icon v-t="['community:discord']" class="mr-1" size="22" color="#ff4600" @click="open('https://www.reddit.com/r/NocoDB/','_blank')">
+              <v-icon v-t="['community:discord']" class="mr-1" size="22" color="#ff4600" @click="open('https://www.reddit.com/r/NocoDB/')">
                 mdi-reddit
               </v-icon>
-              <v-icon v-t="['community:twitter']" class="mr-1" size="22" :color="textColors[1]" @click="open('https://twitter.com/NocoDB','_blank')">
+              <v-icon v-t="['community:twitter']" class="mr-1" size="22" :color="textColors[1]" @click="open('https://twitter.com/NocoDB')">
                 mdi-twitter
               </v-icon>
               <v-icon v-t="['community:book-demo']" class="mr-1" size="22" :color="textColors[3]" @click="open('https://calendly.com/nocodb-meeting','_blank')">
```
Comment (wingkwong) [inline packages/nc-gui/components/project/spreadsheet/components/extras.vue]: oops

Comment (pranavxc) [reply packages/nc-gui/components/project/spreadsheet/components/extras.vue]: updated in next commit

**Comment by ondrejmo** [comment]
Does this mean that swagger will no longer be part of the web interface?

**Pull request merged**